src/test/groovy/org/prebid/server/functional/tests/BidderParamsSpec.groovy

```diff
@@ -32,6 +32,7 @@ import static org.prebid.server.functional.model.request.auction.DistributionCha
 import static org.prebid.server.functional.model.request.auction.DistributionChannel.SITE
 import static org.prebid.server.functional.model.response.auction.ErrorType.PREBID
 import static org.prebid.server.functional.model.response.auction.MediaType.NATIVE
+import static org.prebid.server.functional.testcontainers.Dependencies.getNetworkServiceContainer
```

Comment:
Looks like this shouldn't be here. If I remember correctly there is still a bug in IDEA that creates those imports. And they break the code...